### PR TITLE
Completely removes the SVG from the game and it's variants

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request

This PR completely removes the SVG/AKs from the game including cargo, ruins, loadouts, ships, and the code.

...

...

...

image

... And replaces it with the SKM rifle, a original gun made by me and apoggers! Unlike the Literally A Soviet AK47, the SKM is a original rifle with history rooted in the game's lore and has quirks related to it's origin. This also fixes some of my fuckups:
- .45-70 hollow point boxes now contain 45-70 instead of .357
- 762x39mm for some fucking reason has a bullet spread override that broke spread wielded and unwielded.
- 762x39mm has been renamed 762x40mm CLIP
- the NT-SVG and AK-74 have been merged into the 24v

Also jesus chirst, another 'small' pr turns into a big files changed one and such.

Apogee drew the "Chopper," SKM-44, and SKM 44v

## Why It's Good For The Game

As said before, the SKM actually exists and fits in the game's world and fits the thematic roles the AK served. Also, bless Any and Cole who made the old sprites which have somehow held up for years, but they have shown their age

image

## Changelog

:cl: ryerice, [Apogee-dev](https://github.com/Apogee-dev)
add: All the SVGs have been placed in a Illestren museum, somehow, instead being replaced by the SKM, CLIP's former standard issue rifle!
add: 7.62x39mm has been renamed to 7.62x40mm CLIP
add: LMGs now support bipods; this is a mystery tool that will be useful later!
fix: 7.62x40mm no longer breaks spread on the rifles it's used in
del: NT-SVG and AK-74 have been merged into a single gun
balance: 7.62x40mm now contains eighty rounds per ammo box
fix: 45-70 hollow point ammo boxes now no longer contains 357 for some reason
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -933,10 +933,10 @@
 	item_state = "inteq_webbing"
 	supports_variations = VOX_VARIATION
 
-/obj/item/storage/belt/security/webbing/inteq/ak47/PopulateContents()
+/obj/item/storage/belt/security/webbing/inteq/skm/PopulateContents()
 	. = ..()
 	for(var/i in 1 to 7)
```
Comment (meemofcourse) [inline code/game/objects/items/storage/belt.dm]:
```suggestion
	for(var/i in 1 to 4)
```

**Review by meemofcourse — APPROVED** [review]
rifle is fine

**Comment by Martinpachu** [comment]
awesome sauce

**Comment by Latentish** [comment]
Finally some good fucking removal.

**Comment by rye-rice** [comment]
conflicts fixed

**Pull request merged**